README.md

```diff
@@ -1,5 +1,9 @@
 # LaunchDarkly OpenFeature provider for the Server-Side SDK for PHP
 
+[![CircleCI](https://dl.circleci.com/status-badge/img/gh/launchdarkly/openfeature-php-server/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/launchdarkly/openfeature-php-server/tree/main)
+[![Packagist](https://img.shields.io/packagist/v/launchdarkly/openfeature-server.svg?style=flat-square)](https://packagist.org/packages/launchdarkly/openfeature-server)
+[![Documentation](https://img.shields.io/static/v1?label=GitHub+Pages&message=API+reference&color=00add8)](https://launchdarkly.github.io/openfeature-php-server)
```

Comment:
This is a 404 currently, is that right?

Reply:
Yes. Once I publish the first release, we should be publishing some generated docs there.

Reply:
(Guess so, looks like no docs publish is in the branch.)